auto_submit/lib/service/revert_request_validation_service.dart

```diff
@@ -125,11 +147,24 @@ class RevertRequestValidationService extends ValidationService {
       return;
     }
 
+    final String? revertReason = await getReasonForRevert(githubService, slug, messagePullRequest.number!);
+    if (revertReason == null) {
+      final String message = '''A reason for requesting a revert of ${slug.fullName}/${messagePullRequest.number} could
+      not be found or the reason was not properly formatted. Begin a comment with 'Revert reason:' to tell the bot why
+      this issue is being reverted.''';
```

Comment:
Nit to re-organize the line format to make it look nicer. See https://github.com/flutter/cocoon/pull/3460#issuecomment-1924925501

Also highlight `**Revert reason:**`?

Reply:
Oh good suggestion. I will add it.